raami/queries.js

```diff
@@ -0,0 +1,269 @@
+// var promise = require("bluebird");
+
+// var options = {
+//   // Initialization Options
+//   promiseLib: promise
+// };
+
+// var pgp = require("pg-promise")(options);
+// var db = pgp(process.env.DATABASE_URL)
+
+
+module.exports = {
+  getPeople,
+	getArtists,
+	createArtist,
+	getArtist,
+	updateArtist,
+	getWork,
+	getWorks,
+	createWork,
+	updateWork,
+	removeWork,
+};
+
+function getPeople(req, res, next) {
+  var _id = (req.params.id);                             
+  req.app.locals.db.any(`
+    SELECT id, person_id, name, continent, url, filename, filedata, description,
+           transport, legal, auction, print, digital, agent, contact, waitlist, postage
+      FROM Artist
+     WHERE person_id = $1`, _id
+  )
```

Comment:
Should require `raami_admin` rights, or check that the email address of `person_id` matches `req.session.user.email`.